### PR TITLE
feat: create wrapper library for service_directory

### PR DESCRIPTION
**Pull request by dazuma**

### CODE REVIEW

File: google-cloud-service_directory/.rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+inherit_gem:
+  google-style: google-style.yml
+
+AllCops:
+  Exclude:
+    - "test/**/*"
+
```
Comment (viacheslav-rostovtsev) [inline google-cloud-service_directory/.rubocop.yml]: Rakefile ignored in language but not here

File: google-cloud-service_directory/test/google/cloud/service_directory/client_test.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+require "helper"
+require "google/cloud/service_directory"
+require "gapic/common"
+require "gapic/grpc"
+
+describe Google::Cloud::ServiceDirectory do
+  let(:grpc_channel) { GRPC::Core::Channel.new "localhost:8888", nil, :this_channel_is_insecure }
+
+  it "constructs a secret manager service client with the default version" do
```
Comment (viacheslav-rostovtsev) [inline google-cloud-service_directory/test/google/cloud/service_directory/client_test.rb]: service directory lookup service

File: google-cloud-service_directory/test/google/cloud/service_directory/client_test.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+# limitations under the License.
+
+require "helper"
+require "google/cloud/service_directory"
+require "gapic/common"
+require "gapic/grpc"
+
+describe Google::Cloud::ServiceDirectory do
+  let(:grpc_channel) { GRPC::Core::Channel.new "localhost:8888", nil, :this_channel_is_insecure }
+
```
Comment (viacheslav-rostovtsev) [inline google-cloud-service_directory/test/google/cloud/service_directory/client_test.rb]: add similar test for the RegistrationService?

File: google-cloud-service_directory/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1 @@
+# Release History
```
Comment (viacheslav-rostovtsev) [inline google-cloud-service_directory/CHANGELOG.md]: Do we want to write something like 'initial release of version v1beta1" here?

Comment (dazuma) [reply google-cloud-service_directory/CHANGELOG.md]: The changelog will get updated by releasetool when we do the actual release.

File: google-cloud-service_directory/.rubocop.yml (hunk 1 above)
Comment (dazuma) [reply google-cloud-service_directory/.rubocop.yml]: The exclusion is not needed in this case. We can add it later if it does become necessary.

File: google-cloud-service_directory/test/google/cloud/service_directory/client_test.rb (hunk 2 above)
Comment (dazuma) [reply google-cloud-service_directory/test/google/cloud/service_directory/client_test.rb]: Fixed.

File: google-cloud-service_directory/test/google/cloud/service_directory/client_test.rb (hunk 3 above)
Comment (dazuma) [reply google-cloud-service_directory/test/google/cloud/service_directory/client_test.rb]: Done.

**Pull request merged**